README.md

```diff
@@ -20,16 +14,13 @@ Today we will be making a Tic Tac Toe game using all of this.
 1. Construct a `index.html` to be your starting point on this
    project. Add your necessary HTML tags, including `script` and
```

Comment:
would recommend changing this to seperate `js` and `css` files. Or is there a specific reason we want them to do it all inline?